feature_tests/kt-conf.toml

```diff
@@ -0,0 +1,2 @@
+domain = "dev.gigapixel"
```

Comment:
nit: please use something like `dev.diplomattest`, this feels like a third party package

(I know the domain doesn't exist)

Reply:
also let's call this diplomat-kt-conf.toml